### PR TITLE
Update weird field title for the file_copy_in_dossier-field.

### PR DESCRIPTION
**Pull request by elioschmutz**
This PR updates a weird field title text for the file_copy_in_dossier field.

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -4,6 +4,8 @@ Changelog
 
 4.15.2 (unreleased)
 -------------------
+- Update weird field title for the file_copy_in_dossier-field.
```
Comment (deiferni) [inline docs/HISTORY.txt]: missing newline

**Comment by deiferni** [comment]
oh, ist there an issue for this? if so please also link the issue with a closes #.../fixes #... comment.

**Comment by lukasgraf** [comment]
Ehm, no? `"File a copy"` in english means "eine Kopie ablegen", and is perfectly fine...

**Comment by lukasgraf** [comment]
@elioschmutz http://dict.leo.org/englisch-deutsch/to%20file

**Comment by elioschmutz** [comment]
@deiferni no, there is no issue!

@lukasgraf 😳  thanks! So this PR can be ignored!

**Comment by deiferni** [comment]
oh indeed 👍

**Pull request closed without merging**